utils/calculators.ts

```diff
@@ -31,3 +32,91 @@ export const calcApr = (
 ) => {
   return ((endPrice - startPrice) / startPrice / periodsRemaining) * periods;
 };
+
+// export async function DevMiningEstimate(){
+// }
+type DevMiningCalculatorParams = {
+  ethers: any;
+  getPrice: any;
+  empAbi: any;
+  erc20Abi: any;
+  provider: any;
+};
+export function DevMiningCalculator({
+  provider,
```

Comment:
This is most of the dev mining estimate calculation. Outputs all emps and their estimated rewards.